gammapy/scripts/cta_sensitivity.py

```diff
@@ -0,0 +1,250 @@
+            sensitivity curve is superposed with the one stored in the IRF fits file
+    """
+
+    def __init__(self,irffile,livetime,slope=2.,alpha=0.2,sigma=5.,gamma_min=10.,bkg_sys=0.05, verbosity=0):
+        self.filename = irffile
+        self.livetime = (livetime*u.Unit('h')).to('s')
+        self.slope = slope
+        self.alpha = alpha
+        self.sigma = sigma
+        self.gamma_min = gamma_min
+        self.bkg_sys = bkg_sys
+        self.verbosity = verbosity
+        self.energy = None
+        self.diff_sens = None
+        # Reading of the IRFs
+        self.cta_perf = CTAPerf.read(self.filename)
+
+    def get_bkg(self, bkg_rate):
+        bkg = bkg_rate.data.data * self.livetime
+        return bkg.value * u.Unit('')
+
+    # Cf. np.vectorize for an time optimisation of this function
+    # For the moment, search by dichotomy
+    def get_excess(self, bkg_counts):
+
+        excess = np.zeros(len(bkg_counts))
+        for icount in range(len(bkg_counts)):
+    
+            # Coarse search
+            start, stop = -1., 6.
+            coarse_excess = np.logspace(start=start, stop=stop, num=1000)
+            coarse_on = coarse_excess + bkg_counts[icount]
+            coarse_off = np.zeros(len(coarse_on)) + bkg_counts[icount] / self.alpha
+            coarse_sigma = significance_on_off(n_on=coarse_on, n_off=coarse_off, alpha=self.alpha, method='lima')
+            idx = np.abs(coarse_sigma - self.sigma).argmin()
+        
+            start = coarse_excess[max(idx-1,0)]
+            stop = coarse_excess[min(idx+1, len(coarse_sigma)-1)]
+            #if self.verbosity == 2:
+            #    print('# {}, {}'.format(start, stop))
+            #    print('{}, {}'.format(idx, len(coarse_sigma)-1))
+            if start == stop:
+                print('LOGICAL ERROR> Impossible to find a number of gamma!')
+                excess[icount] = -1
+                continue
+        
+            # Finer search
+            num = int((stop - start)/0.1)
+            fine_excess = np.linspace(start=start, stop=stop, num=num)
+            fine_on = fine_excess + bkg_counts[icount]
+            fine_off = np.zeros(len(fine_on)) + bkg_counts[icount] / self.alpha
+            fine_sigma = significance_on_off(n_on=fine_on, n_off=fine_off, alpha=self.alpha, method='lima')
+            idx = np.abs(fine_sigma - self.sigma).argmin()
+            if fine_excess[idx] >= self.gamma_min and fine_excess[idx] >= self.bkg_sys * bkg_counts[icount]:
+                excess[icount] = fine_excess[idx]
+            else:
+                excess[icount] = max(self.gamma_min, self.bkg_sys * bkg_counts[icount])
+
+            if self.verbosity > 0 :
+                print('N_ex={}, N_fineEx={}, N_bkg={}, N_bkgsys={}, Sigma={}'.format(excess[icount],fine_excess[idx], \
+                    bkg_counts[icount],self.bkg_sys * bkg_counts[icount],fine_sigma[idx]))
+
+        return excess
+
+    # Return differential fluxes
```

Comment:
Instead of putting `# comments` like that, put a docstring on the method.
Comments are lost when Python parses your file, but docstrings are stored on the class / method / function as the `__doc__` attribute and are then shown in the online HTML docs and the interactive python `help` system.
Every "public" function or method should have a one-line docstring.
See examples at http://docs.gammapy.org/en/latest/development/howto.html#documentation-guidelines